consensus/istanbul/core/core.go

```diff
@@ -37,6 +37,12 @@ import (
 	"github.com/rcrowley/go-metrics"
 )
 
+const (
+	// The number in exceptional case
+	// If the number of validators is 6 or less, the chain could be forked by round change
+	ExceptionalValidatorsNumber = 6
```

Comment:
I changed the comment like below, is it right? btw, this variable is only used at `roundchange.go`. What about moving to that file?
```suggestion
	// If the number of validators is 6 or less, the chain could be forked by round change if the required minimum consensus message is "2f+1". 
	// So, the exceptional case such as number of validator is 6, gather more messages than "2f+1". See requiredMessageCount for more specific information.
	exceptionalValidatorsNumber = 6
```

Reply:
Thanks for your review.
I reflected in [da28338689ac05293ca5691987eec457622b0d9d](https://github.com/klaytn/klaytn/pull/1462/commits/da28338689ac05293ca5691987eec457622b0d9d).